frappe/public/js/frappe/form/save.js

```diff
@@ -130,10 +131,18 @@ frappe.ui.form.save = function (frm, action, callback, btn) {
 						folded = frm.layout.folded;
 					}
 
+					if (df.fieldtype === 'Tab Break') {
+						tab = df;
+					}
+
 					if (is_docfield_mandatory(doc, df) &&
 						!frappe.model.has_value(doc.doctype, doc.name, df.fieldname)) {
 						has_errors = true;
 						error_fields[error_fields.length] = __(df.label);
+
+						// show tab
+						frm.layout.tabs_dict[tab.fieldname].set_active();
+
 						// scroll to field
 						if (!frm.scroll_set) {
 							scroll_to(doc.parentfield || df.fieldname);
```

Comment:
We should instead be fixing `scroll_to` so we dont have to make this change in all the places? 👀

Reply:
this *might* simplify supporting tabs in UI tests too.

Reply:
this feature already exists 🙈

doctype was not configured correctly

Reply:
https://github.com/frappe/frappe/blob/develop/frappe/public/js/frappe/form/form.js#L1775

Thanks for the review :-)